google/cloud/datastore/client.py

```diff
@@ -784,28 +784,6 @@ def do_something(entity):
 
         or manually page through results
 
-        .. testsetup:: query-page
-
-            import os
-            import uuid
-
-            from google.cloud import datastore
-            from tests.system.test_system import Config  # system tests
-
-            unique = os.getenv('CIRCLE_BUILD_NUM', str(uuid.uuid4())[0:8])
-            client = datastore.Client(namespace='ns{}'.format(unique))
-
-            key = client.key('_Doctest')
-            entity1 = datastore.Entity(key=key)
-            entity1['foo'] = 1337
-            entity2 = datastore.Entity(key=key)
-            entity2['foo'] = 42
-            Config.TO_DELETE.extend([entity1, entity2])
-            client.put_multi([entity1, entity2])
-
-            query = client.query(kind='_Doctest')
-            cursor = None
-
```

Comment:
Note to reviewers:  I've ripped out the bit-rotted doctest setup / teardown code here and below (we haven't had a `doctests` session since https://github.com/googleapis/google-cloud-python/pull/6642, 2018-11-28).  #209 tracks the need to get the docstring doctests either tested, or else replaced by snippets / samples.